src/ZipkinOpenTracing/Span.php

```diff
@@ -23,10 +23,6 @@ final class Span implements OTSpan
 
     private array $remoteEndpointArgs;
 
-    private ?Scope $scope = null;
```

Comment:
Scope knows about the span but span should not know about the scope. In this case also we transfer the responsibility of closing the scope to the span but on Tracer::startActiveSpan provide a scope that we expect the user to close. This represents a cumbersome user experience and also wrong.